Cargo.toml

```
@@ -9,28 +9,29 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
-bytes = "0.5"
+bytes = "1.0"
 futures-core = "0.3"
 futures-util = { version = "0.3", default-features = false }
 rand = "0.7"
 log = "0.4.1"
 httparse = "1.3.4"
 http = "0.2.0"
 twoway = "0.2.1"
-tokio = { version = "0.2", optional = true, features= ["fs", "stream"] }
-tokio-util = { version = "0.3", optional = true, features= ["codec"] }
+tokio = { version = "1.0", optional = true, features= ["fs"] }
+tokio-util = { version = "0.6", optional = true, features= ["codec"] }
 mime_guess = {version = "2.0.1", optional = true}
 thiserror = "1.0"
 pin-project = "0.4.22"
 
 [dev-dependencies]
-hyper = "0.13"
-warp = "0.2.2"
+hyper = "0.14"
+warp = { git = "https://github.com/aknuds1/warp", branch = "chore/upgrade-tokio" }
```

```suggestion
warp = { git = "https://github.com/seanmonstar/warp" }
```

The PR branch was removed after the merging. Not sure if this can create an issue, probably if other changes arise.

---

```suggestion
warp = { default-features = false, version = "0.3" }
```

It'd seem that default features are `multipart` and `websocket` which should be unused by the example.